Source/Core/VideoBackends/OGL/OGLTextureRaw.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "VideoBackends/OGL/OGLTextureRaw.h"
+
+namespace OGL
+{
+OGLTextureRaw::OGLTextureRaw(u32 stride, u32 width, u32 height, std::vector<u8>&& glData)
+    : AbstractRawTexture(nullptr, stride, width, height), m_glData(std::move(glData))
```

Comment:
@lioncash - sorry to bother you but this is specifically what I was trying to decide on.  If you have time, please take a look and see what you think.  In particular, I wasn't sure how bad it was to pass nullptr and then set it below in the body.